sim/midas/src/main/cc/bridges/termination.h

```diff
@@ -0,0 +1,38 @@
+
+#ifndef __TERMINATION_H
+#define __TERMINATION_H
+//See LICENSE for license details.
+
+#ifdef TERMINATIONBRIDGEMODULE_struct_guard
+
+#include "bridge_driver.h"
+
+class termination_t: public bridge_driver_t
+{
+  public:
+    termination_t(simif_t* sim, 
+    std::vector<std::string> &args,
+    TERMINATIONBRIDGEMODULE_struct * mmio_addrs,
+    unsigned int num_messages,
+    unsigned int* isErr,
+    const char* const* msgs);
+    ~termination_t();
+    virtual void init() {};
+    virtual void tick();
+    virtual void finish() {};
+    virtual bool terminate() { return test_done; };
+    virtual int exit_code() { return fail; };
+    const char* exit_message();
+    int cycleCount();
```

Comment:
```suggestion
    int cycle_count();
```

Snake case on class methods.